### PR TITLE
IO-380 adding nginx init script to playbook

### PR DESCRIPTION
**Pull request by alexandarp**

### CODE REVIEW

File: files/etc/init.d/nginx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        stop)
+                echo -n "Stopping $DESC: "
+                start-stop-daemon --stop --quiet --pidfile /var/run/nginx/nginx.pid \
+                    --exec $DAEMON || true
+                echo "$NAME."
+                ;;
+
+        restart|force-reload)
+                echo -n "Restarting $DESC: "
+                if ! test_nginx_config; then
```
Comment (alexandarp) [inline files/etc/init.d/nginx]: @pdufault Should I add a console message here?

Something like the following:
```
echo -n "Checking Config for $DESC: "
```

**Comment by stevenharradine** [comment]
is there no default script to start with?  I was expecting one line to be moved in an existing script

**Comment by stevenharradine** [comment]
Comment withdrawn after in person discussion

**Comment by aaronpederson** [comment]
Team Approved

**Pull request merged**